pkg/kubelet/certificate/transport.go

```diff
@@ -91,7 +93,7 @@ func updateTransport(stopCh <-chan struct{}, period time.Duration, clientConfig
 		//
 		// See: https://github.com/kubernetes-incubator/bootkube/pull/663#issuecomment-318506493
 		t.closeAllConns()
-	}, period, stopCh)
+	}, 0, stopCh)
```

Comment:
no 0 waits... that's a hotloop if the rotate channel gets closed

Reply:
the channel should not be closed in any case by hand. Do you mean need process exceptional close here?